### PR TITLE
Fixes instruction page for the assignment

### PR DESCRIPTION
**Pull request by annagav**
- Disabled arrow buttons if there is only one instruction.
- Changed the instruction text of the sutd2 assignment.

### CODE REVIEW

**Comment by pdpinch** [comment]
It's Ok that these are combined in one PR, since they are part of the same view. But I would have preferred to have them separate as your 1st bullet is a template change for all assignments and your 2nd bullet is a content change that only applies to sutd2.

**Comment by pdpinch** [comment]
Whoa, why so many changes to AssignmentsView.js ?

**Comment by annagav** [comment]
I did code reformat to fix some of the alignment I ruined.

File: html_app/master_model.js [hunk 1]
```diff
@@ -16942,9 +16942,9 @@ var fix_assignment_models =  function (assignment_result) {
 		}
 		if(assignment_result[x].id == 'sutd_intro_biology'){
 			sutd_intro_biology = assignment_result[x];
-			sutd_intro_biology.template.instructions = [
-				['Getting Started', 'Please refer to your instructor for detailed instructions.']
-				];
+//			sutd_intro_biology.template.instructions = [
+//				['Getting Started', 'Please refer to your instructor for detailed instructions.']
+//				];
```
Comment (pdpinch) [inline html_app/master_model.js]: Can we just delete these lines?

File: html_app/ui/assignments.soy [hunk 2]
```diff
@@ -78,19 +78,21 @@ Assignment description
...
 				{$section[1] |noAutoescape}</div>   
-        	{/foreach} 
+        	{/foreach}
+        	{if length($assignments.selected.template.instructions)>1}
         	<div class='scb_s_assignments_bottom_scroll'>
    				<div class = 'scb_s_assignment_header_img_left scb_assignments_bottom_arrow_left scb_s_assignment_header_img_left_inactive'  assignment_id='{$assignments.selected.id}'   role='button' aria-label='Previous Section'></div>
    				<div class='scb_s_assignments_button_divider'></div>
 				<div class = 'scb_s_assignment_header_img_right scb_assignments_bottom_arrow_right'  assignment_id='{$assignments.selected.id}'  role='button' aria-label='Next Section'></div>
 			</div>
+			{/if}
```
Comment (pdpinch) [inline html_app/ui/assignments.soy:95]: I think this would be easier to read if these two lines (94-95) were de-indented to match lines 89-90.

**Comment by pdpinch** [comment]
:+1: once the comments are addressed.

File: html_app/master_model.js (hunk 1 above)
Comment (annagav) [inline html_app/master_model.js]: That's right. Ill do it now.

**Pull request merged**